src/parse-css.js

```diff
@@ -167,7 +167,7 @@ function parseCss(css, options = {}) {
         match(/^([;\s]*)+/); // ignore empty declarations + whitespace
 
         const comment_regexp = /\/\*[^*]*\*+([^/*][^*]*\*+)*\//g;
-        let prop = match(/^(\*?[-#/*\\\w]+(\[[0-9a-z_-]+\])?)\s*/);
+        let prop = match(/^(\*?[-#\/*\\\w\.]+(\[[0-9a-z_-]+\])?)\s*/);
```

Comment:
The two added backslash characters are unnecessary. Only the period after the `w` is required.